.packit.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       centos-stream-9:
         additional_repos:
           - https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/
-          - https://download.copr.fedorainfracloud.org/results/@centos-automotive-sig/bluechi-snapshot/centos-stream-9-x86_64/
+          - https://download.copr.fedorainfracloud.org/results/@centos-automotive-sig/qm-snapshot/centos-stream-9-x86_64/
 
```

Comment:
AFAIR this url for bluechi was for package dependency for packit
Why, did you change it ?